sdk/eng/mgmt/mgmtmetadata/sql_resource-manager.txt

```diff
@@ -0,0 +1,15 @@
+﻿Installing AutoRest version: 2.0.4413
+AutoRest installed successfully.
+Commencing code generation
```

Comment:
maybe we need to add this file in eng/mgmt/mgmtmetadata/sql_resource-manager.txt ? 

FYI this most recent merged SQL PR: https://github.com/Azure/azure-sdk-for-net/pull/27036/files

Reply:
and please rebase your forked repo/branch to the latest main branch